src/main/webapp/charts.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html>
+
+    <head>
+        <title>Charts</title>
+        <meta charset="UTF-8">
+        <link rel="stylesheet" href="/css/main.css">
+        <link rel="stylesheet" href="/css/user-page.css">
+        <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+        <script type="text/javascript" src="/js/chart-loader.js"></script>
+    </head>
+
+    <body onload="buildUI();">
+        <nav>
+            <ul id="navigation">
+                <li><a href="/">Home</a></li>
+                <li><a href="/aboutus.html">About Our Team</a></li>
+                <li><a href="/feed.html">Public Feed</a></li>
+                <li><a href="/stats.html">Stats</a></li>
+            </ul>
+        </nav>
+
+        <h1 id="page-title">Chart Page</h1>
+
+        <div id="chart"></div>
+
+    </body>
+
+</html>
```

Comment:
I think adding an extra line at the end of the file will resolve the github no newline problem